samples/subsys/modbus/rtu_server/app.overlay

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2020 Phytec Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_serial {
+	status = "okay";
+
+	modbus0 {
+		compatible = "modbus,rtu";
```

Comment:
Could also be modbus-rtu, since modbus is not the manufacturer.

Reply:
Yeah that is kind of not right, I am looking at it now.

Reply:
renamed to zephyr,modbus-serial